readme.md

```diff
@@ -0,0 +1,30 @@
+![Polylang](assets/polylang-logo.png)
+
+Welcome to the Polylang repository on GitHub. Here you can browse the source, look at open
+issues and keep track of development.
+
+If you are not a developer, please use the [Polylang plugin page](https://wordpress.org/plugins/polylang/) on [WordPress.org](https://wordpress.org).
+
+## [Pre-requisites](#pre-requisites)
+
+Before starting, make sure you have the following software installed and working on your machine:
+
+1. A local [WordPress](https://wordpress.org/support/article/how-to-install-wordpress/) (5.1 or later) instance
+2. [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) to clone the Polylang repository (or your fork of the Polylang repository).
+3. [NVM](https://github.com/nvm-sh/nvm) or [chocolatey](https://chocolatey.org/install) (on Windows OS) to install [Node.js](https://nodejs.org/en/download/) and [NPM](https://docs.npmjs.com/). They are required by [Webpack](https://webpack.js.org/guides/getting-started/) that Polylang uses to build and minifies CSS and javascript files.
+4. [Composer](https://getcomposer.org/doc/00-intro.md) because Polylang uses its autoloader to work and it is required to install development tools such as PHP CodeSniffer that ensures your code follows code standards.
+
+## [How to set up Polylang](#how-to-setup-polylang)
+
+The most simple way is to clone locally this repository directly in your local WordPress instance `wp-content/plugins/` folder.
```

Comment:
This is the first step below. Do we need to repeat it?

Reply:
It isn't not really the same. It is to precise it is a way simple and to get started quickly but there are other ways.
However I recognize the sentence is restrictive to only the first step.

I'm going to change it for a more general one to introduce the process to follow.